README.md

```diff
@@ -1 +1,172 @@
-# Portfolio-setup-and-mobile-first
\ No newline at end of file
+<a name="readme-top"></a>
+
+# Portfolio-setup-and-mobile-first
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio-setup-and-mobile-first] <a name="about-project"></a>
+
+> The first step in a long project to come.
+
+**[Portfolio-setup-and-mobile-first]** is the toughest challenge I've yet faced.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
[OPTIONAL] These placeholders are targeting you, not the visitors. So please replace them with your text 👍